clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -1675,6 +1675,15 @@ public static <V> Map<String, V> entriesWithPrefix(Map<String, V> map, String pr
         return result;
     }
 
+    /**
+     * Checks requirement. Throw {@link IllegalArgumentException} if {@code requirement} failed.
+     * @param requirement Requirement to check.
+     */
+    public static void require(boolean requirement) {
```

Comment:
Can it take custom error message? Current error message is a bit meaningless.

Reply:
My goal it to rewrite classes as is, but OK let's fix this :)